tests/importResolver.test.ts

```diff
@@ -0,0 +1,64 @@
+import { ImportMapResolver } from "../src/importMapResolver.ts";
+import { assertEquals } from "./deps.ts";
+
+Deno.test("importResolver", async (t) => {
+  const importMap = {
+    imports: {
+      "react": "https://esm.sh/react",
+      "fmt/": "https://deno.land/std@0.134.0/fmt/",
+      "/": "./",
+      "./": "./",
+      "app.tsx": "./src/app.tsx",
+      "#/": "./src/",
+    },
+  };
+
+  function assertFileHrefEquals(href: string, expected: string) {
+    return assertEquals(href, `file://${Deno.cwd()}/${expected}`);
```

Comment:
Should this be updated to `file://${Deno.cwd()}/tests/${expected}`? As this would match the import.meta.url?

Reply:
Ah yes, I was in that directory when I was running the tests.